packages/internet_detector.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,37 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'internet_detector'
+$category = 'Utilities'
+$packageToolDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+# Create tool directory
+$toolDir = Join-Path ${Env:RAW_TOOLS_DIR} $toolName
+New-Item -Path $toolDir -ItemType Directory -Force -ea 0
+VM-Assert-Path $toolDir
+
+# Install dependency for windows api
+VM-Pip-Install "pywin32"
+# Install dependency for creating python executable
+VM-Pip-Install "pyinstaller"
+
+Start-Process -FilePath 'cmd.exe' -ArgumentList "/c pyinstaller --onefile -w --distpath $toolDir --workpath $packageToolDir --specpath $packageToolDir $packageToolDir\internet_detector.pyw" -Wait
+
+# Move images to %VM_COMMON_DIR% directory
+$imagesPath = Join-Path $packageToolDir "images"
+Copy-Item "$imagesPath\*" ${Env:VM_COMMON_DIR} -Force
+
+VM-Install-Shortcut -toolName $toolName -category $category -executablePath "$toolDir/$toolName.exe"
+
+# TODO - Uncomment when FakeNet BlackList for DNS is fixed/addressed. https://github.com/mandiant/flare-fakenet-ng/issues/190
+# # Create scheduled task for tool to run every 2 minutes.
+# $action = New-ScheduledTaskAction -Execute $rawToolPath
+# $trigger = New-ScheduledTaskTrigger -Once -At (Get-Date) -RepetitionInterval (New-TimeSpan -Minutes 2)
+# Register-ScheduledTask -Action $action -Trigger $trigger -TaskName 'Internet Detector' -Force
+
+# Copy updated `default.ini` with tool added to ProcessBlackList to FakeNet Config Directory
```

Comment:
Alternatively (better more robust solution): Move this to our fakenet package in this PR.

We can still open the PR/issue in fakenet afterwards to discuss if porting the change upstream is an option, as otherwise we will have to keep this file in sync in our package which is not ideal.